test/e2e/specs/editor/blocks/links.spec.js

```diff
@@ -8,6 +8,537 @@ test.describe( 'Links', () => {
+				},
+			},
+		] );
+	} );
+
+	test( `can be removed`, async ( { page, editor, LinkUtils } ) => {
+		await LinkUtils.createAndReselectLink();
+
+		// Unlick via shortcut
+		// we do this to avoid an layout edge case whereby
+		// the rich link preview popover will obscure the block toolbar
+		// under very specific circumstances and screensizes.
+		await page
+			.locator( '.block-editor-link-control__search-item-top' )
+			.getByRole( 'button', { name: 'Unlink' } )
+			.click();
+
+		// The link should have been removed.
+		await expect.poll( editor.getBlocks ).toMatchObject( [
+			{
+				name: 'core/paragraph',
+				attributes: {
+					content: 'This is Gutenberg',
+				},
+			},
+		] );
+	} );
+
+	test( `allows Left to be pressed during creation when the toolbar is fixed to top`, async ( {
+		page,
+		editor,
+		pageUtils,
+		LinkUtils,
+	} ) => {
+		await LinkUtils.toggleFixedToolbar( true );
+
+		// Create a block with some text.
+		await editor.insertBlock( {
+			name: 'core/paragraph',
+		} );
+		await page.keyboard.type( 'Text' );
+		await page.getByRole( 'button', { name: 'Link' } ).click();
+
+		// Typing "left" should not close the dialog.
+		await pageUtils.pressKeys( 'ArrowLeft' );
+		let popover = page
+			//TODO: change to a better selector when https://github.com/WordPress/gutenberg/issues/51060 is resolved.
+			.locator(
+				'.components-popover__content .block-editor-link-control'
+			);
+		await expect( popover ).toBeVisible();
+
+		// Escape should close the dialog still.
+		await page.keyboard.press( 'Escape' );
+		popover = page
+			//TODO: change to a better selector when https://github.com/WordPress/gutenberg/issues/51060 is resolved.
+			.locator(
+				'.components-popover__content .block-editor-link-control'
+			);
+		await expect( popover ).not.toBeVisible();
+	} );
+
+	test( `allows Left to be pressed during creation in "Docked Toolbar" mode`, async ( {
+		page,
+		editor,
+		pageUtils,
+		LinkUtils,
+	} ) => {
+		await LinkUtils.toggleFixedToolbar( false );
+
+		// Create a block with some text.
+		await editor.insertBlock( {
+			name: 'core/paragraph',
+		} );
+		await page.keyboard.type( 'Text' );
+
+		await editor.clickBlockToolbarButton( 'Link' );
+
+		// Typing "left" should not close the dialog.
+		await pageUtils.pressKeys( 'ArrowLeft' );
+		let popover = page
+			//TODO: change to a better selector when https://github.com/WordPress/gutenberg/issues/51060 is resolved.
+			.locator(
+				'.components-popover__content .block-editor-link-control'
+			);
+		await expect( popover ).toBeVisible();
+
+		// Escape should close the dialog still.
+		await page.keyboard.press( 'Escape' );
+		popover = page
+			//TODO: change to a better selector when https://github.com/WordPress/gutenberg/issues/51060 is resolved.
+			.locator(
+				'.components-popover__content .block-editor-link-control'
+			);
+		await expect( popover ).not.toBeVisible();
+	} );
+
+	test( `can be edited with collapsed selection`, async ( {
```

Comment:
```suggestion
	test( `can be edited when within a link but no selection has been made ("collapsed")`, async ( {
```

Let's be clear about the behaviour and avoid technical jargon.